internal/jennies/terraform/templates/types/models.tmpl

```diff
@@ -0,0 +1,14 @@
+package provider
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+{{ $package := .Schema.Package | formatObjectName }}
+{{ range .Objects | }}
+type {{ formatModelName $package .Name }}DataSourceModel struct {
+  {{ range .Type.Struct.Fields | filterScalars }}
```

Comment:
I'd remove the `| filterScalars` thing and make `formatTerraformType` return `any` for types we don't handle yet.